messages/rest.md

```diff
@@ -0,0 +1,36 @@
+# summary
+
+Makes an authenticated HTTP request to the Salesforce REST API and prints the response.
+
+# description
+
+You must specify a Salesforce org to use, either with the --target-org flag or by setting your default org with
```

Comment:
Do we need this sentence?  I assume --target-org will be a required flag?  I don't think we explicitly say this for other commands that require an org, such as "project deploy start".